src/controller/articlesController.js

```diff
@@ -137,6 +137,35 @@ class ArticleController {
     }
   }
 
+  /**
+   *
+   * @param {object} req - request object
+   * @param {object} res - response object
+   * @returns {object} - server response
+   */
+  static async getArticleById(req, res) {
+    try {
+      const { articleId } = req.params;
+      const article = await Article.findByPk(articleId);
+      if (article) {
+        const views = article.dataValues.viewStats;
+        await article.update({
+          viewStats: views + 1
+        }, { hooks: false });
+        return res.status(200).json({
```

Comment:
We also need to create a class controller for error responses, as the `status 200` messages are repeated several times over the whole project.